### PR TITLE
guestbook v2

### PR DESCRIPTION
**Pull request by szihai**

### CODE REVIEW

File: v2/guestbook-v2-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: guestbook
+        version: "2.0"
+    spec:
+      containers:
+      - name: guestbook
+        image: linsun/guestbook:v2
```
Comment (linsun) [inline v2/guestbook-v2-deployment.yaml]: should be pointing to the ibm repo.

**Comment by linsun** [comment]
we already have this https://github.com/IBM/guestbook/blob/master/v2/guestbook-deployment.yaml

File: v2/guestbook-v2-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
```
Comment (duglin) [inline v2/guestbook-v2-deployment.yaml]: Why the new filename?

Comment (cmluciano) [reply v2/guestbook-v2-deployment.yaml]: -1 we should use the apps deployment path

**Comment by duglin** [comment]
@szihai can we close this?

**Comment by linsun** [comment]
In istio101 there is a plan dir

**Pull request closed without merging**